src/style.css

```diff
@@ -1 +1,73 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  --max__height: 60px;
+  --active_color_link: #00acdc;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  max-height: var(--max__height);
+  box-sizing: border-box;
+}
+
+.logo {
+  display: block;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  line-height: 14px;
+  display: flex;
+}
+
+.nav__element_list-item {
+  list-style: none;
+  margin: 0 10px;
+}
+
+.nav__element_list-item:last-child {
+  margin-right: 0;
+}
+
+.nav__element_list-item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  display: flex;
+  color: #000;
+  text-decoration: none;
+  height: var(--max__height);
+  line-height: var(--max__height);
+}
```

Comment:
1) you can just use display: inline-block since you don't use flex as it usually used
2) it's not good to hardcode height for element with content inside. Use `padding: 23px 0;` instead

Reply:
I'm using display: flex; cos i wanna to put my line inder link, if i use inline-block - line shifts to the right and becomes, as it were, after the text. Like that:
![image](https://github.com/mate-academy/layout_moyo-header/assets/72886995/0c695d2c-d5ef-4c14-a1e3-4d0d696e93e6)

I didn't find a better solution than to leave flex and set a specific height for the entire header. Btw, one mentor in the chat said that i can use height: 100%, so i just add max-height in a variable and can use it wherever i want.